src/components/AddTodoForm.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react'; 
+import PropTypes from 'prop-types';
+import InputWithLabel from './InputWithLabel';
+
+function AddTodoForm({ onAddTodo }) {
+  const [todoTitle, setTodoTitle] = useState('');
+
+  const handleTitleChange = (event) => {
+    const newTodoTitle = event.target.value;
+    setTodoTitle(newTodoTitle);
+  };
+  function handleAddTodo(event) {
+    event.preventDefault();
+
+    setTodoTitle('');
+    onAddTodo(todoTitle);
+  }
+
+  return (
+    <form onSubmit={handleAddTodo}>
+      <div className="input-container">
+        <InputWithLabel
+          id="todoTitle"
+          handleTitleChange={(event) => handleTitleChange(event)}
+          todoTitle={todoTitle}
+          isFocused
+        >
+          Title
+        </InputWithLabel>
+        <button type="submit" disabled={todoTitle === ''}>Add</button>
```

Comment:
this is another situation where using Truthy/Falsy can simplify a condition: 

```suggestion
        <button type="submit" disabled={!todoTitle}>Add</button>
```
Because `Boolean("")` evaluates to `false` (try it in your browser console), `!todoTitle` includes the empty string possibility as well as `undefined`, `null` and any other Falsy value.